src/message/html/HtmlTagImg.js

```diff
@@ -10,10 +13,32 @@ const styles = StyleSheet.create({
   },
 });
 
-export default ({ src, style, auth }) => (
-  <Image
-    source={getResource(src, auth)}
-    resizeMode="contain"
-    style={[styles.img, style]}
-  />
-);
+class HtmlTagImg extends Component {
+  constructor(props) {
+    super(props);
+    this.onPress = this.onPress.bind(this);
+
+    const { src, auth } = this.props;
+    this.fullUrl = getResource(src, auth);
```

Comment:
yeah, this was a typo, I had meant to add it to props. But using it directly in render certainly makes sense.